### PR TITLE
Script to re-label Safari 12.1 STP runs

### PR DESCRIPTION
**Pull request by stephenmcgruer**
See discussion in
https://github.com/web-platform-tests/wpt.fyi/issues/1175#issuecomment-564586992

### CODE REVIEW

**Comment by stephenmcgruer** [comment]
@Hexcles PTAL. I don't intend to land this, but want it reviewed before I run it. On dry-run, it hits just under 150 runs.

File: tagger/safari_12_1_stp.go [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+func (e safari12STPLabeller) ShouldProcessRun(run *shared.TestRun) bool {
+	if run.BrowserName != "safari" {
+		return false
+	}
+	if !strings.HasPrefix(run.BrowserVersion, "12.1") {
+		return false
+	}
+	// We ran Safari 12.1 as STP from 2018-08-07 until 2019-01-05. It then
+	// released as stable 2019-04-09. So anything after February (to pick
```
Comment (foolip) [inline tagger/safari_12_1_stp.go:27]: What is the earliest run after this cutoff point?

**Review by foolip — APPROVED** [review]

File: tagger/safari_12_1_stp.go (hunk 1 above)
Comment (stephenmcgruer) [reply tagger/safari_12_1_stp.go:27]: Sorry, a bit of misleading wording.

Safari 12.1 was released on 2019/03/25 (https://en.wikipedia.org/wiki/Safari_version_history#Safari_12). The first stable WPT run (aka the earliest run) was 2019/04/09.

**Comment by stephenmcgruer** [comment]
Script has been run on staging + prod. Logs attached:

[stp_12_1_log.txt](https://github.com/web-platform-tests/data-migration/files/4017621/stp_12_1_log.txt)
[stp_12_1_log_staging.txt](https://github.com/web-platform-tests/data-migration/files/4017622/stp_12_1_log_staging.txt)

**Pull request closed without merging**